inspire/modules/workflows/tasks/submission.py

```diff
@@ -86,11 +87,17 @@ def _create_ticket(obj, eng):
         rt_queue = cfg.get("CFG_BIBCATALOG_QUEUES") or queue
         sip = d.get_latest_sip(sealed=False)
         subject = u"Your suggestion to INSPIRE: {0}".format(d.title)
+        try:
```

Comment:
`ifilter` returns an iterator, `next()` gets only the first element because having more than one comment with the `'submitter'` source would break it... Is this possible to happen? Should we prevent it from breaking the task?

@jmartinm @jalavik, WDYT?